docs/layout.md

```diff
@@ -1,49 +1,40 @@
-Navigating Remix
-================
-Remix IDE is comprised of three panels and a terminal.
+# Navigating Remix
 
-![](images/a-layout1c.png)
+Remix IDE consists of four core components: the icon panel, side panel, main panel, and terminal. Since Remix v0.69.0, it also includes a top bar.
 
-1. Icon Panel - click to change which plugins appear in the Side Panel
-2. Side Panel - most but not all plugins have their interface here
-3. Main Panel - for editing files, large format tools, and the home tab
-4. Terminal - for viewing transaction receipts and various logs
+![Layout for Remix IDE, highlighting the terminal, topbar, and, the side, main, and icon panel.](images/layout/a-layout1c.png)
 
-Default Tools
------------------------
+- Icon Panel - portal to most of Remix's core functions like the Solidity compiler, plugins, and deploy & run.
+- Top Bar - interface for workspace selection, GitHub authentication, themes, and Remix configuration settings.
+- Side Panel - interface for the file explorer, compiler, search page, deploy & run page, and certain plugins.
+- Main Panel - for editing files, large format tools, and the home tab.
```

Comment:
Main Panel - contains tabs for editing files, Main Panel plugins (plugins that require more space than Side Panel plugins), and the Home Tab.